src/main/webapp/app/overview/course-overview.component.ts

```diff
@@ -194,15 +207,99 @@ export class CourseOverviewComponent implements OnInit, OnDestroy, AfterViewInit
         await this.initAfterCourseLoad();
         this.sidebarItems = this.getSidebarItems();
         this.courseActionItems = this.getCourseActionItems();
+        this.updateVisibility(window.innerHeight);
+        this.updateMenuOffset();
+    }
+
+    /** Listen window resizement event by height */
+    @HostListener('window: resize', ['$event'])
+    onResize() {
+        this.dropdownOpen = false;
+        this.dropdownClickNumber = 0;
+        this.updateVisibility(window.innerHeight);
+        this.updateMenuOffset();
+    }
+
+    /** Listen click event on anywhere outside of the dropdown menu */
+    @HostListener('document: click', ['$event'])
+    onClickOutsideDropdownMenu() {
+        if (this.dropdownOpen) {
+            this.dropdownClickNumber += 1;
+            if (this.dropdownClickNumber === 2) {
+                this.dropdownOpen = false;
+                this.dropdownClickNumber = 0;
+            }
+        }
```

Comment:
This function does not only handle outside of the menu clicks, but also clicks for the items in the dropdown menu as well. Since "More" label is **outside of the dropdownContent**, it makes "this.dropdownOpen" property **true** after clicking "More" label in the navigation bar, which results in closing the dropdown menu immediately, which is the unwanted behaviour. Therefore, your suggestion is currently not applicable.